src/components/reset-password-2fa-sms/tests/reset-password-2fa-sms-controller.test.ts

```diff
@@ -59,6 +59,57 @@ describe("reset password 2fa auth app controller", () => {
 
       expect(res.render).to.have.calledWith("reset-password-2fa-sms/index.njk");
     });
+
+    it("should render security code entered too many times page view when user is account is locked from entering security codes", async () => {
```

Comment:
I've noticed that in some tests you say it should render a given view, but in others you stipulate the the Nunjucks file. For example, here you say: 

> should render **security code entered too many times page view** when user is account is locked from entering security codes

and in a later test you say 

> should render **security-code-error/index-wait.njk** when user was locked out in the current session for requesting too many security codes

I'm wondering if it might be worth consistently using the `.njk` variant so it's clear we're expecting a given template rather than some conditionally included parts of the template.